SOSS/dms/soss_centroids.py

```diff
@@ -321,7 +394,7 @@ def get_uncontam_centroids_edgetrig(image, header=None, mask=None, poly_order=11
     if mask is None:
         mask = np.zeros_like(image, dtype='bool')
 
-    # Call the script that determines the dimensions of the stack.
+    # Call the script that determines the dimensions of the image.
     dimx, dimy, xos, yos, xnative, ynative, padding, refpix_mask = \
         get_image_dim(image, header=header, verbose=verbose)
```

Comment:
I don't like the `\` operator 
you could do this:
```python
gout = get_image_dim(image, header=header, verbose=verbose)
dimx, dimy, xos, yos, xnative, ynative, padding, refpix_mask = gout
```

or if line 2 is still too long:
```python
gout = get_image_dim(image, header=header, verbose=verbose)
dimx, dimy, xos, yos, xnative = gout[:5]
ynative, padding, refpix_mask = gout[5:]
```